custom_components/foxess_modbus/entities/modbus_integration_sensor.py

```diff
@@ -0,0 +1,109 @@
+"""Sensor"""
+import logging
+from dataclasses import dataclass
+
+from custom_components.foxess_modbus.const import FRIENDLY_NAME
+from custom_components.foxess_modbus.entities.modbus_entity_mixin import (
+    ModbusEntityMixin,
+)
+from homeassistant.components.integration.sensor import IntegrationSensor
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.components.sensor import SensorEntityDescription
+from homeassistant.components.sensor import SensorStateClass
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfTime
+from homeassistant.helpers.entity import Entity
+
+from ..common.entity_controller import EntityController
+from .entity_factory import EntityFactory
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(kw_only=True)
+class ModbusIntegrationSensorDescription(SensorEntityDescription, EntityFactory):
+    """Custom sensor description"""
+
+    address: int
```

Comment:
I don't think we use the address for anything. Can we remove it (and return an empty list from `addresses`)